appinventor/components/src/com/google/appinventor/components/runtime/BluetoothConnectionBase.java

```diff
@@ -76,7 +77,7 @@ private BluetoothConnectionBase(Form form, String logTag) {
    * This constructor is for testing purposes only.
    */
   protected BluetoothConnectionBase(OutputStream outputStream, InputStream inputStream) {
-    this((Form) null, (String) null);
+    this((Form) null, (String) null, SdkLevel.LEVEL_CUPCAKE);
```

Comment:
Since the earliest SDK version we target is 7, you should use `LEVEL_ECLAIR_MR1` rather than `LEVEL_CUPCAKE`